helm/hpcc/templates/dfuserver.yaml

```diff
@@ -91,10 +91,12 @@ spec:
 {{- include "hpcc.addVolumeMounts" $commonCtx | indent 8 }}
 {{ include "hpcc.addConfigMapVolumeMount" . | indent 8 }}
 {{ include "hpcc.addCertificateVolumeMount" (dict "root" $ "name" .name "component" "dfuserver" "external" false) | indent 8 }}
+{{- include "hpcc.addSecretVolumeMounts" $commonCtx | nindent 8 }}
       volumes:
 {{ include "hpcc.addConfigMapVolume" . | indent 6 }}
 {{- include "hpcc.addVolumes" $commonCtx | indent 6 }}
 {{ include "hpcc.addCertificateVolume" (dict "root" $ "name" .name "component" "dfuserver" "external" false) | indent 6 }}
+{{- include "hpcc.addSecretVolumes" $commonCtx | indent 6 }}
```

Comment:
is {{- to chomp whitespace ok here? (e.g. if there are no secrets).
I may be, but notice elsewhere that addSecretVolumes is included there is no leading white space chomping.

Reply:
It was ok to remove the {{-.  There is an extra blank line but that's caused by hpcc.addCertificateVolume.